blog/2015-09-30-HowLuckyAreYourRandomSeeds.adoc

```diff
@@ -0,0 +1,140 @@
+*** Remove the `FIRST_FIT_DECREASING` solver benchmark (`<solverBenchmark>` element).
+** MachineReassignment:
+*** Add `<subSingleCount>8</subSingleCount>` to the inherited solver benchmark configuration.
+*** Remove the `original` and `lateAcceptance1000` solver benchmark (`<solverBenchmark>` elements).
+
+* Each run takes about 10 hours.
+* Every planning problem runs for at least 5 minutes. Every run starts with a 30 second JVM warm up which is discarded.
+
+* Other VM arguments: `-Xmx8192m` +
+Oracle JDK version: `1.7.0_79` +
+Software: `Linux version 4.1.6-201.fc22.x86_64` +
+Hardware: `Intel(R) Core(TM) i7-4810MQ CPU @ 2.80GHz`
+
+* To reproduce these benchmarks locally, http://www.optaplanner.org/code/sourceCode.html[build OptaPlanner from source]
+and run the classes
+https://github.com/droolsjbpm/optaplanner/blob/master/optaplanner-examples/src/main/java/org/optaplanner/examples/app/GeneralOptaPlannerBenchmarkApp.java[GeneralOptaPlannerBenchmarkApp],
+https://github.com/droolsjbpm/optaplanner/blob/master/optaplanner-examples/src/main/java/org/optaplanner/examples/curriculumcourse/app/CurriculumCourseBenchmarkApp.java[CurriculumCourseBenchmarkApp]
+and
+https://github.com/droolsjbpm/optaplanner/blob/master/optaplanner-examples/src/main/java/org/optaplanner/examples/machinereassignment/app/MachineReassignmentBenchmarkApp.java[MachineReassignmentBenchmarkApp]
+after applying the configuration changes described above.
+
+== Benchmark results
+
+You can view the individual benchmark reports at the following links:
+
+* https://docs.jboss.org/optaplanner/blog/benchmark/2015-09-30/general[General benchmark report]
+** This benchmark solves one or two datasets of different use-cases with only 1 solver configuration per use-case.
+** There were no differences in hard scores between runs.
+** In the problem benchmark score distribution charts of runs,
+we can see that there are no outliers (they would be drawn as circles in the charts).
+** The relative percentage change from worst to best soft score
+`(|worstScore|-|bestScore|)/|bestScore|)` was at most 13% in Curriculum Course (`comp08`).
+
+* https://docs.jboss.org/optaplanner/blog/benchmark/2015-09-30/curriculumcourse[Curriculum Course benchmark report]
+** This benchmark solves 7 datasets of the same use-case with 2 different solver configurations.
+** There were differences in hard score between runs (`comp02`, `comp05`, `comp06`) -- this use-case is prone to them.
+*** Specifically, Tabu Search on this use-case is prone to finding an infeasible solution - Late Acceptance only had 5
+infeasible runs (4 of them on the `comp05` dataset and 1 on `comp02`) as compared to Tabu Search's 13 infeasible runs.
+** There were no outliers on the hard score level.
+** There were 2 outliers in runs of the Late Acceptance solver configuration:
+the relative percentage change in `comp07` being 25%, in `comp01` 104%.
```

Comment:
Thanks, published :)

One more thing to think about: Why `(57-5)/5` and not `(57-5)/57` ? :)
The phrase "the relative percentage change" is ambiguous, because it doesn't note the base.
Here's how I would state it:
  The best subsingle is 91% better than the worst subsingle. (`(57-5)/57`)
or
  The worst subsingle is 1040% better than the best subsingle. (`(57-5)/5).

Actually, `10.4 != 104%`, it's `1040%`.